typescript/web/src/components/website/Blog/BlogCard.tsx

```diff
@@ -40,7 +41,8 @@ export const BlogCard = (props: BlogProps) => {
           height="60"
           objectFit="cover"
           alt={title}
-          src={image ?? "/static/img/home-screenshot2.jpg"}
+          // FIXME The fallback image (initially home-screenshot2.jpg) does not exists
+          src={image ?? getImageUrl("home-screenshot-2.jpg")}
```

Comment:
@geoffreyvancassel here is a defect I've found while changing the URLs (see FIXME above)